src/test/java/org/wildfly/test/integration/vdx/utils/server/ServerDomain.java

```diff
@@ -62,24 +62,24 @@ public void startServer() {
 
     @Override
     public void archiveModifiedUsedConfig() throws Exception {
-        Files.copy(Paths.get(DOMAIN_CONFIGURATION_PATH.toString(), getServerConfig().configuration()),
+        Files.copy(Paths.get(CONFIGURATION_PATH.toString(), getServerConfig().configuration()),
```

Comment:
archiveModifiedUsedConfig() looks the same in ServerDomain.java and ServerStandalone.java and can be moved to ServerBase as well. It's duplicated code.